### PR TITLE
Fragment/Frame/Group dropdown: convert to/from Group

### PR DESCRIPTION
**Pull request by balazsbajorics**
**Problem:**
The Frame / Fragment / Group dropdown's Group option was disabled.

**Fix:**
I am re-enabling the group conversion with a few tests.

### CODE REVIEW

File: editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.tsx [hunk 1]
```diff
@@ -246,89 +247,6 @@ export function absoluteResizeBoundingBoxStrategy(
   }
 }
 
-function createResizeCommandsFromFrame(
```
Comment (balazsbajorics) [inline editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.tsx:249]: this was moved verbatim to `resize-strategy-helpers.ts`

File: editor/src/components/inspector/editor-contract-section.spec.browser2.tsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+import { selectComponentsForTest, wait } from '../../utils/utils.test-utils'
 import type { EditorContract } from '../canvas/canvas-strategies/strategies/contracts/contract-helpers'
 import { mouseClickAtPoint } from '../canvas/event-helpers.test-utils'
 import type { EditorRenderResult } from '../canvas/ui-jsx.test-utils'
 import { getPrintedUiJsCode, renderTestEditorWithCode } from '../canvas/ui-jsx.test-utils'
 import { groupSectionOption } from './editor-contract-section'
 
 const projectWithSizedDiv = `import * as React from 'react'
-import { Storyboard } from 'utopia-api'
+import { Storyboard, Group } from 'utopia-api'
```
Comment (Rheeseyb) [inline editor/src/components/inspector/editor-contract-section.spec.browser2.tsx:11]: I understand why you have added `Group` to the imports here, but please can we have a test the ensures the import is automatically added

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/components/inspector/editor-contract-section.spec.browser2.tsx (hunk 2 above)
Comment (balazsbajorics) [reply editor/src/components/inspector/editor-contract-section.spec.browser2.tsx:11]: sure!

Comment (balazsbajorics) [reply editor/src/components/inspector/editor-contract-section.spec.browser2.tsx:11]: done in 54eef1672

**Review by ruggi — APPROVED** [review]

**Pull request merged**